src/controllers/resources/form/collections.js

```diff
@@ -86,7 +86,7 @@ export default function () {
     const { products } = Data()
     if (products && products.length) {
       const query = `_id:("${products.join('" "')}")`
-      callSearchApi(`items.json?q=${encodeURIComponent(query)}`, 'GET', function (err, data) {
+      callSearchApi(`items.json?q=${encodeURIComponent(query)}&size=${products.length}`, 'GET', function (err, data) {
```

Comment:
```suggestion
      callSearchApi(`items.json?q=${encodeURIComponent(query)}&size=${Math.min(products.length, 100)}`, 'GET', function (err, data) {
```
Não conferi onde exatamente isso está indo, mas provavelmente tem que ter um limite aqui até pra não travar o browser do usuário...

Reply:
Estava mostrando somente 10, mas faz sentido, muitos não pode